_episodes_rmd/03-raster-reproject-in-r.Rmd

```diff
@@ -214,49 +206,50 @@ UTM. However, the extent values of `DTM_hillUTMZ18N_HARV` are different from
 > 
 > Why do you think the two extents differ?
 > > ## Answers
-> > ```{r challenge-code-extent-crs, echo=TRUE}
-> > # The extent for DTM_hill_UTMZ18N_HARV is in UTMs so the extent is in meters.
-> > # The extent for DTM_hill_HARV is still in lat/long so the extent is expressed
-> > # in decimal degrees.
+> > The extent for DTM_hill_UTMZ18N_HARV is in UTMs so the extent is in meters. The extent for DTM_hill_HARV is still in lat/long so the extent is expressed
+> > in decimal degrees.
 > > ```
 > {: .solution}
 {: .challenge}
 
 ## Deal with Raster Resolution
 
-Let's next have a look at the resolution of our reprojected hillshade.
+Let's next have a look at the resolution of our reprojected hillshade versus our original data.
 
 ```{r view-resolution}
-
-# compare resolution
 res(DTM_hill_UTMZ18N_HARV)
-
+res(DTM_hill_HARV)
 ```
 
-The output resolution of `DTM_hill_UTMZ18N_HARV` is 1 x 0.998. Yet, we know that
-the resolution for the data should be 1m x 1m. We can tell `R` to force our
+These two resolutions are different, but they're representing the same data. We can tell `R` to force our
 newly reprojected raster to be 1m x 1m resolution by adding a line of code
-(`res=`).
+(`res=`) within the `projectRaster()` function.
 
 ``` {r reproject-assign-resolution }
-# adjust the resolution
 DTM_hill_UTMZ18N_HARV <- projectRaster(DTM_hill_HARV,
                                   crs = crs(DTM_HARV),
                                   res = 1)
-# view resolution
-res(DTM_hill_UTMZ18N_HARV)
-
 ```
 
-Let's plot our newly reprojected raster.
+Let's double-check our resolution to be sure
+```{r}
+res(DTM_hill_UTMZ18N_HARV)
+res(DTM_hill_HARV)
+```
 
-```{r plot-projected-raster }
-# convert the reprojected raster to a data.frame()
+Now both our resolutions and our CRSs match, so we can plot these two
+data sets together. For plotting with `ggplot()`, we will need to
+create a dataframe from our newly reprojected raster.
 
-DTM_hill_HARV_2_df <- DTM_hill_UTMZ18N_HARV  %>% 
+```{r make-df-projected-raster}
+DTM_hill_HARV_2_df <- DTM_hill_UTMZ18N_HARV  %>%
```

Comment:
this can be simplified with `raster::as.data.frame(rasterobj, xy = TRUE)`

Reply:
Great! Glad to hear it can be simplified. We'll need to do that in several of the episodes, so I'll put in a separate issue to remind myself to look for all those places and change them in one go.

Reply:
https://github.com/datacarpentry/r-raster-vector-geospatial/issues/194